src/main/kotlin/parts/wisdom/arcraider/SimpleLineWorld.kt

```diff
@@ -0,0 +1,42 @@
+package parts.wisdom.arcraider
+
+import me.joypri.Part
+import me.joypri.to
+
+/**
+ *
+ * This file is just an example of how to use our building blocks.
+ */
+
+
+val myGrid = Grid(
+        Height to 5,
+        Width to 3
+)
+
+val myLine = Line(
+        TheColor to Color.RED,
+        TheDirection to Direction.RIGHT,
+        TheCoords to Coords(X to 2, Y to 3),
+        Length to 1
+)
+
+open class Generator(vararg parts: Part) : ArcMix(*parts) {
+    val grid by TheGrid
+    val line by TheLine
+
+    fun makeGrid(): DavidsGrid {
```

Comment:
For `List<List<Foo>>`, I think of the outer list as containing rows, and I think of that as the standard approach. If so, I think `grid.width` and `grid.height` should be swapped in the `return` expression.